SPECS/nvme-cli/nvme-cli.spec

```diff
@@ -1,18 +1,17 @@
 Name:           nvme-cli
 Summary:        NVM-Express user space tooling for Linux
 Version:        1.16
-Release:        1%{?dist}
+Release:        2%{?dist}
 Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 License:        GPLv2
 URL:            https://github.com/linux-nvme/nvme-cli
-Source0:        https://github.com/linux-nvme/%{name}/archive/%{name}-%{version}.tar.gz
-#Source0:       https://github.com/linux-nvme/%{name}/archive/v%{version}.tar.gz
+Source0:        https://github.com/linux-nvme/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 
-%if %{with_check}
-BuildRequires:  python3-pip
-%endif
+BuildRequires:  gcc
+BuildRequires:  make
+Requires(post): util-linux systemd systemd-udev
```

Comment:
```suggestion
Requires(post): systemd
Requires(post): systemd-udev
Requires(post): util-linux
```

Reply:
Thanks, will update this along with some other linter suggestions